composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "magento2-module",
     "license": "MIT",
     "require": {
-        "omise/omise-php": "dev-master"
```

Comment:
@guzzilar What is the reason to specific the version of a dependency, Omise-PHP, to 2.6.0?